examples/api-samples/src/browser/chat/chat-node-toolbar-action-contribution.ts

```diff
@@ -0,0 +1,41 @@
+// *****************************************************************************
+// Copyright (C) 2024 STMicroelectronics and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import {
+    ChatNodeToolbarActionContribution
+} from '@theia/ai-chat-ui/lib/browser/chat-node-toolbar-action-contribution';
+import {
+    isResponseNode,
+    RequestNode,
+    ResponseNode
+} from '@theia/ai-chat-ui/lib/browser/chat-tree-view';
+import { interfaces } from '@theia/core/shared/inversify';
+
+export function bindChatNodeToolbarActionContribution(bind: interfaces.Bind): void {
+    bind(ChatNodeToolbarActionContribution).toDynamicValue(context => ({
+        getToolbarActions: (args: RequestNode | ResponseNode) => {
+            if (isResponseNode(args)) {
+                return [{
+                    commandId: 'core.about',
+                    icon: 'codicon codicon-feedback',
+                    tooltip: 'Show about dialog'
```

Comment:
I would like to suggest the following, this makes the example intent much more clear
```suggestion
                    commandId: 'sample-command',
                    icon: 'codicon codicon-feedback',
                    tooltip: 'Example command'
```